scipy/optimize/_nonlin.py

```diff
@@ -1320,8 +1320,8 @@ class KrylovJacobian(Jacobian):
     %(params_basic)s
     rdiff : float, optional
         Relative step size to use in numerical differentiation.
-    method : {'lgmres', 'gmres', 'bicgstab', 'cgs', 'minres'} or function
-        Krylov method to use to approximate the Jacobian.
+    method : {'lgmres', 'gmres', 'bicgstab', 'cgs', 'minres', 'tfqmr'} or
+        callable Krylov method to use to approximate the Jacobian.
```

Comment:
This is still wrong, `callable belongs on the line above. I pushed a commit to fix it, and a code comment to explain.

Reply:
Sorry, I only considered PEP8 at that time, so I put `callable` on the next line. The sentence `Krylov method to use to approximate the Jacobian` should really start from another line.